app/src/hooks/DbListenerInit/syncingNodeListener.ts

```diff
@@ -248,34 +249,63 @@ export const invokeSyncingIfRequired = async ({
   appMode?: "EXTENSION" | "DESKTOP";
   isSyncEnabled?: boolean;
 }): Promise<void> => {
-  if (!uid || (!team_id && !isSyncEnabled)) return;
+  const transaction = startTransaction({ name: "recordsSyncing" });
```

Comment:
This won't track complete transaction from starting fetching the rules from firebase to storing them in StorageService. This will only track the transaction after rules has been fetched from firebase